src/main/scala/loamstream/loam/LoamGraph.scala

```diff
@@ -236,18 +246,20 @@ final case class LoamGraph(
       storeConsumers = storeConsumersNew)
   }
 
-  def nameOf(t: Tool): Option[String] = namedTools.collectFirst { case (n, namedTool) if namedTool == t => n }
+  def nameOf(t: Tool): Option[String] = namedTools.get(t)
```

Comment:
Very nice. 👍

Reply:
Thanks, that felt _very_ good to do.  That `collectFirst` bit was a hack that wasn't a problem until it was.

Reply:
> ...that wasn't a problem until it was.

Like all code. :-)